app/utils/QueryStringUtils.scala

```diff
@@ -3,7 +3,6 @@ package utils
 
 object QueryStringUtils {
   def addFormDesignTestParameterQueryString(queryString: Map[String, Seq[String]]): Map[String, Seq[String]] = {
-    val formDesignTestParameterValue = if (Random.nextFloat <= 0.2) "control" else "variant"
-   queryString + ("formDesignTest" -> Seq(formDesignTestParameterValue))
```

Comment:
This change has actually already happened. annoyingly, I checked out this branch from a branch where I'd previously made this change. Don't worry it won't cause any problems, I promise.